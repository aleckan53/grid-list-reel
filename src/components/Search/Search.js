import React from 'react';
import styles from './Search.module.css';

export default function Search() {
  return (
    <div className={styles.search}>
      <input type='text' className={styles.bar}/>
    </div>
  )
}