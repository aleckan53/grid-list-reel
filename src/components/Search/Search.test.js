import React from 'react';
import ReactDOM from 'react-dom';
import { mount } from 'enzyme';
import Search from './Search';

describe('Search component', () => {

  it('matches the snapshot', () => {
    const root = document.createElement('div');
    ReactDOM.render(<Search/>, root);
    expect(root).toMatchSnapshot();
    ReactDOM.unmountComponentAtNode(root);
  })
})