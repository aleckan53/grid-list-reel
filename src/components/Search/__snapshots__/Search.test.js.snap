// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Search component matches the snapshot 1`] = `
<div>
  <div
    class="search"
  >
    <input
      class="bar"
      type="text"
    />
  </div>
</div>
`;
